decidim-core/app/packs/stylesheets/decidim/application.scss

```diff
@@ -1,8 +1,83 @@
+  @import "stylesheets/decidim/_accordion.scss";
+  @import "stylesheets/decidim/_tos.scss";
+  @import "stylesheets/decidim/_vertical_tabs.scss";
+  @import "stylesheets/decidim/_flash.scss";
+  @import "stylesheets/decidim/_static-map.scss";
+  @import "stylesheets/decidim/_filters.scss";
+  @import "stylesheets/decidim/_tags.scss";
+  @import "stylesheets/decidim/_tabs_x.scss";
+  @import "stylesheets/decidim/_metadata.scss";
+  @import "stylesheets/decidim/_modal.scss";
+  @import "stylesheets/decidim/_modal_share.scss";
+  @import "stylesheets/decidim/_modal_flag.scss";
+  @import "stylesheets/decidim/_modal_identities.scss";
+  @import "stylesheets/decidim/_modal_authorization.scss";
+  @import "stylesheets/decidim/_modal_update.scss";
+  @import "stylesheets/decidim/_modal_tos_refuse.scss";
+  @import "stylesheets/decidim/_modal_fingerprint.scss";
+  @import "stylesheets/decidim/_address.scss";
+  @import "stylesheets/decidim/_statistics.scss";
+  @import "stylesheets/decidim/_content_blocks.scss";
+  @import "stylesheets/decidim/_documents.scss";
+  @import "stylesheets/decidim/_floating_help.scss";
+  @import "stylesheets/decidim/_cards.scss";
+  @import "stylesheets/decidim/_floating_help.scss";
+  @import "stylesheets/decidim/_metrics.scss";
+  @import "stylesheets/decidim/_order-by.scss";
+  @import "stylesheets/decidim/_photos.scss";
+  @import "stylesheets/decidim/_endorsers_list.scss";
+  @import "stylesheets/decidim/_activity.scss";
+  @import "stylesheets/decidim/_tooltip.scss";
+  @import "stylesheets/decidim/_author.scss";
+  @import "stylesheets/decidim/_order-by.scss";
+  @import "stylesheets/decidim/_wizard_steps.scss";
+  @import "stylesheets/decidim/_progress-bar.scss";
+  @import "stylesheets/decidim/_spinner.scss";
+  @import "stylesheets/decidim/_toggle_switch.scss";
+  @import "stylesheets/decidim/_hero.scss";
+  @import "stylesheets/decidim/_actions.scss";
+}
+
+// On the other hand, the following styles match with specific routes
+@import "stylesheets/decidim/_header.scss";
+@import "stylesheets/decidim/_footer.scss";
+@import "stylesheets/decidim/_login.scss";
+@import "stylesheets/decidim/_pages.scss";
+@import "stylesheets/decidim/_notifications.scss";
+@import "stylesheets/decidim/_profile.scss";
+@import "stylesheets/decidim/_conversations.scss";
+@import "stylesheets/decidim/_versions.scss";
+@import "stylesheets/decidim/_home.scss";
+@import "stylesheets/decidim/_search.scss";
+@import "stylesheets/decidim/_cookies.scss";
+// participatory spaces are quite similar between processes and assemblies (different modules), so their stylesheets are placed in the core
+@import "stylesheets/decidim/_participatory_spaces.scss";
+@import "stylesheets/decidim/_omnipresent_banner.scss";
 
 // This imports all the Decidim module stylesheet imports registered at
 // `config/assets.rb` of the module for the "app" group.
-@import "!decidim-style-imports[app]";
+// @import "!decidim-style-imports[app]";
```

Comment:
why do you disable this ?

Reply:
That was disabled over a year long. Now the stylesheets are being imported via `append_stylesheets_pack_tag` when are required, and barely there are modules using that clause.

Reply:
Disabling this would break modules that want to customize the Decidim default layout without having to add `append_stylesheet_pack_tag` to all of the views. Such modules as [`mainio/decidim-theme-bleach`](https://github.com/mainio/decidim-theme-bleach). Although this module might not be relevant after the redesign is completed but similar modules might become handy.

It was mentioned before by @furilo before that we would not break existing behavior. There is further discussion about this at #11191.